src/js/structs/Batch.js

```diff
@@ -42,7 +42,7 @@
 
 class Batch {
   constructor() {
-    const batch = [];
+    const batch = [{action: 'INIT'}];
```

Comment:
This is a solution which I also considered to have the batch.reduce execute at least once. But I do think this is more a workaround then a solution to the problem. We should consider to rewrite the reduce function like that:

```JS
reduce(callback, data){
  if (this.length === 0) {
    return callback(data);
  }
  return this.reduce(callback, data);
}
```

In that case we are executing the callback at least once. And we are saving one cycle of `combineReducers` which does not any `value`.